lib/connection/connection.js

```diff
@@ -265,12 +266,15 @@ function Connection(context) {
    * @returns {Object} the connection object.
    */
   this.connectAsync = async function (callback) {
+    const connectingDomain = connectionConfig.accessUrl && connectionConfig.accessUrl.includes('snowflakecomputing.cn') ? 'CHINA' : 'GLOBAL';
```

Comment:
can we create a method to not duplicate the logic and log?

Reply:
Fixed